createBrainMask.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+
+from dipy.segment.mask import median_otsu
+
+
+def createBrainMaskFromb0Data(b0Data):
+    medianRadius = 4
+    numIterations = 4
```

Comment:
Add a function description to function and of function arguments. Why MedianRadius = 4?